.github/CODEOWNERS

```diff
@@ -1,755 +1,773 @@
 
 # ServiceLabel: %Cognitive - Bing %Service Attention
-/sdk/cognitiveservices/Search.Bing*            @jaggerbodas-ms @arwong
+/sdk/cognitiveservices/Search.Bing*                                @jaggerbodas-ms @arwong
 
 # ServiceLabel: %Cognitive - Mgmt %Service Attention
-/sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/            @yangyuan
+/sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/ @yangyuan
 
 # ServiceLabel: %Commerce %Service Attention
-#/<NotInRepo>/            @ms-premp @qiaozha
+#/<NotInRepo>/                                                     @ms-premp @qiaozha
 
 # PRLabel: %Communication
-/sdk/communication/                                     @acsdevx-msft
+/sdk/communication/                                                @acsdevx-msft
 
 # PRLabel: %Communication - Calling Server
-/sdk/communication/Azure.Communication.CallingServer/   @minwoolee-msft
+/sdk/communication/Azure.Communication.CallingServer/              @minwoolee-msft
 
 # PRLabel: %Communication - Chat
-/sdk/communication/Azure.Communication.Chat/   @LuChen-Microsoft
+/sdk/communication/Azure.Communication.Chat/                       @LuChen-Microsoft
 
 # PRLabel: %Communication - Common
-/sdk/communication/Azure.Communication.Common/          @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft @mjafferi-msft
+/sdk/communication/Azure.Communication.Common/                     @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft @mjafferi-msft
 
 # PRLabel: %Communication - Identity
-/sdk/communication/Azure.Communication.Identity/        @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft @mjafferi-msft
+/sdk/communication/Azure.Communication.Identity/                   @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft @mjafferi-msft
 
 # PRLabel: %Communication - Network Traversal
-/sdk/communication/Azure.Communication.NetworkTraversal/ @ajpeacock0 @nathpete-msft
+/sdk/communication/Azure.Communication.NetworkTraversal/           @ajpeacock0 @nathpete-msft
 
 # PRLabel: %Communication - Phone Numbers
-/sdk/communication/Azure.Communication.PhoneNumbers/    @miguhern @whisper6284 @RoyHerrod @danielav7
+/sdk/communication/Azure.Communication.PhoneNumbers/               @miguhern @whisper6284 @RoyHerrod @danielav7
 
 # PRLabel: %Communication - Short Codes
-/sdk/communication/Azure.Communication.ShortCodes/      @guilhermeluizsp @danielav7
+/sdk/communication/Azure.Communication.ShortCodes/                 @guilhermeluizsp @danielav7
 
 # PRLabel: %Communication - SMS
-/sdk/communication/Azure.Communication.Sms/             @RoyHerrod @arifibrahim4
+/sdk/communication/Azure.Communication.Sms/                        @RoyHerrod @arifibrahim4
 
+# TODO: OWNERS MISSING HERE!
 # PRLabel: %Communication - Resource Manager
 /sdk/communication/Azure.ResourceManager.Communication/
```

Comment:
@jsquire FYI owners missing here. See:
- https://github.com/Azure/azure-sdk-for-net/pull/27760

Reply:
That's a management library.    It's Arthur Ma and Elendil Zheng.

Reply:
All. *.ResourceManager.* are those folks.

Reply:
OK, this path is superseded by `/**/Azure.ResourceManager*/` below anyway. I am thus going to remove it.